physicsnemo/distributed/manager.py

```diff
@@ -311,7 +311,7 @@ def initialize_env():
         rank = int(os.environ.get("RANK"))
         world_size = int(os.environ.get("WORLD_SIZE"))
         if "LOCAL_RANK" in os.environ:
-            local_rank = os.environ.get("LOCAL_RANK")
+            local_rank = int(os.environ.get("LOCAL_RANK"))
```

Comment:
Isn't this int conversion handled in the next `if` block? And won't this break when `LOCAL_RANK` is `None`?

Reply:
can that happen? if LOCAL_RANK is set, can it still be None: like with 

export LOCAL_RANK=

?